test/render/test-cases/h3-layers.js

```diff
@@ -1,7 +1,13 @@
 import {H3HexagonLayer, H3ClusterLayer} from '@deck.gl/geo-layers';
+import GL from '@luma.gl/constants';
 
 import * as h3 from 'h3-js';
 
+const parameters = {
```

Comment:
I do not think you need to add culling to `H3HexagonLayer` and `HexagonLayer`. They eventually render the `ColumnLayer`. In the spirit of unit tests, the `ColumnLayer` render tests should include:

1. flat with stroke
2. extruded with wireframe

Each combined with

a. default disk geometry
b. externally provided vertices with CCW winding order
c. externally provided vertices with CW winding order

Reply:
I see. The issue with winding appeared only for hexbin layer, not for h3, probably because hexbin used the default disk geometry, and such a case wasn't covered.

Also, I would enable culling for all layers where we expect culling to work. Render tests are a good place to catch such culling issues.